node/rpc.go

```diff
@@ -0,0 +1,133 @@
+package node
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/harmony-one/harmony/core"
+	"github.com/harmony-one/harmony/internal/hmyapi"
+)
+
+const (
+	rpcHTTPPortOffset = 10
+	rpcWSPortOffset   = 20
+)
+
+var (
+	// HTTP RPC
+	rpcAPIs      []rpc.API
+	httpListener net.Listener
+	httpHandler  *rpc.Server
+
+	wsListener net.Listener
+	wsHandler  *rpc.Server
+
+	httpEndpoint = ""
+	wsEndpoint   = ""
+
+	httpModules      = []string{"hmy"}
```

Comment:
const?

Reply:
we may able to modify enabled modules at run time and restart RPC to provide more modules. so not using const.